client/src/resources/ItemStore.js

```diff
@@ -133,113 +133,82 @@ export default class ItemStore {
+        searchParams.append("publication", filter.selected);
       }
       if (filter.name === "active" && filter.selected !== "all") {
-        queries.push(
-          `active:${filter.selected === "onlyActive" ? "true" : "false"}`
-        );
+        if (filter.selected === "onlyActive") {
+          searchParams.append("active", "true");
+        } else {
+          searchParams.append("active", "false");
+        }
       }
     }
-
+    if (onlyTools) {
+      searchParams.append("tool", JSON.stringify(onlyTools));
+    }
     if (searchString) {
-      queries.push(
-        `(id:${searchString}* OR title:${searchString}* OR subtitle:${searchString}* OR annotations:${searchString}*)`
-      );
+      searchParams.append("searchString", searchString);
     }
-
-    if (onlyTools) {
-      let onlyToolsQuery = onlyTools
-        .map(tool => {
-          return `tool:"${tool}" OR tool:${tool.replace(
-            new RegExp("_", "g"),
-            "-"
-          )}`;
-        })
-        .join(" OR ");
-
-      queries.push(`(${onlyToolsQuery})`);
+    if (limit) {
+      searchParams.append("limit", limit);
     }
-
-    let body = {
-      include_docs: true,
-      limit: limit || 18,
-      sort: "-orderDate"
-    };
-
-    if (queries.length > 0) {
-      body.query = queries.join(" AND ");
-    } else {
-      body.query = "*:*";
+    if (bookmark) {
+      searchParams.append("bookmark", bookmark);
     }
-    return body;
+    const QServerBaseUrl = await qEnv.QServerBaseUrl;
+    return `${QServerBaseUrl}/search?${searchParams.toString()}`;
   }
 
   async getItems(
     searchString = undefined,
-    limit,
+    limit = 18,
     onlyTools = undefined,
     bookmark = undefined
   ) {
-    let searchRequestBody = await this.getSearchRequestBody(
+    const searchUrl = await this.getSearchUrl(
       searchString,
       limit,
+      bookmark,
       onlyTools
     );
-    if (bookmark) {
-      searchRequestBody.bookmark = bookmark;
-    }
-    return this.loadItems(searchRequestBody);
-  }
-
-  async loadItems(searchRequestBody) {
-    const QServerBaseUrl = await qEnv.QServerBaseUrl;
-    const response = await this.httpClient.fetch(`${QServerBaseUrl}/search`, {
-      method: "POST",
-      body: JSON.stringify(searchRequestBody)
+    const response = await this.httpClient.fetch(searchUrl, {
+      method: "GET"
     });
 
     if (!response.ok) {
       throw response;
     }
 
     const data = await response.json();
-    if (!data.rows) {
-      data.rows = [];
+    if (!data.docs) {
+      data.docs = [];
     }
 
-    const items = data.rows.map(row => {
+    const items = data.docs.map(doc => {
       let item = new Item(this.user, this.httpClient);
-      item.addConf(row.doc);
-      this.items[row.doc._id];
+      item.addConf(doc);
+      this.items[doc._id];
       return item;
     });
-
     return {
       items: items,
-      total_rows: data.total_rows,
+      moreItemsAvailable: data.docs.length === limit,
```

Comment:
please add a comment explaining this for our future selfs :-)

Reply:
Yes sure -> https://github.com/nzzdev/Q-editor/pull/97/commits/571147d29afdc35a077c49cce1355e3a2299385f